caip-XX.md

```diff
@@ -0,0 +1,88 @@
+---
+caip: <to be assigned>
+title: Browser Session Token Model
+author: <a list of the author's or authors' name(s) and/or username(s), or name(s) and email(s), e.g. (use with the parentheses or triangular brackets): Olaf Tomalski (@ritave), Hassan BM (@hbm-88), and Juan Caballero (@bumblefudge)
+discussions-to: <URL>
+status: Draft
+type: Standard
+created: 2022-11-08
+updated: 2022-11-08
+requires (*optional): <CAIP number(s)>
+replaces (*optional): <CAIP number(s)>
+---
+
+<!--You can leave these HTML comments in your merged EIP and delete the visible duplicate text guides, they will not appear and may be helpful to refer to if you edit it again. This is the suggested template for new EIPs. Note that an EIP number will be assigned by an editor. When opening a pull request to submit your EIP, please use an abbreviated title in the filename, `eip-draft_title_abbrev.md`. The title should be 44 characters or less.-->
+
+## Simple Summary
+<!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the CAIP.-->
+This data model proposes a minimal set of properties that structure a browser-based wallet<>dapp session for interoperability and security purposes.
```

Comment:
It probably makes sense to define this as a protocol so security of the session can be analyzed more easily rather than a data model.